src/dev-dashboard/ui/src/slate-grid.css

```
@@ -523,28 +579,109 @@
     box-shadow: 0 0 9px rgba(52, 211, 153, 0.35);
 }
 .dd-strip {
+    --dd-strip-gutter: 36px;
+    display: grid;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)